undercloud.yml

```diff
@@ -39,6 +96,14 @@
         template:
             src: undercloud.conf.j2
             dest: "{{ undercloud_conf }}"
+
+      - name: Set overcloud_domain_name in undercloud.conf
+        lineinfile:
+          path: "{{ undercloud_conf }}"
```

Comment:
can you customize overcloud_domain_name in the template  undercloud.conf.j2 based on tls is enabled/disabled

Reply:
Ack